src/backend/api/reviews.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+//Returns all reviews
+router.get("/", async (request, response) => {
+  const allReviews = await knex("review").select("*");
+  response.json(allReviews);
+});
+
+//Adds a new review
+router.post("/", async (request, response) => {
+  try {
+    const newReview = await knex("review").insert(request.body);
+    response.json(newReview);
+  } catch (error) {
+    throw error;
```

Comment:
If you're **only** throwing the error again, there is no need to use try/catch.
Perhaps it would be useful to console.log the error first and then throw, or throw a custom error that can be displayed for the user or instead send a error code back with `response.status(503)`